docs/architecture/overview.md

```diff
@@ -7,21 +7,26 @@ This document specifies the architecture for GCP Ingestion as a whole.
 ![diagram.mmd](diagram.svg "Architecture Diagram")
 
 - The Kubernetes `Ingestion Edge` sends messages from `Producers` (e.g.
-  Firefox) to PubSub `Raw Topics`
-- The `Raw Sink` job copies messages from PubSub `Raw Topics` to
+  Firefox) to a set of PubSub `Raw Topics`, routing messages based on `uri`
+- `Raw Topics` are the first layer of a "pipeline family"; the diagram shows
+  only the "structured" pipeline family, but there are also deployments
+  for "telemetry", "stub-installer", and "pioneer"
+- The `Raw Sink` job copies messages from a PubSub `Raw Topic` to
   `BigQuery`
-- The Dataflow `Decoder` job decodes messages from PubSub `Raw Topics` to
-  PubSub `Decoded Topics`
-    - The Dataflow `Decoder` job checks for existence of `document_id`s in
+- The Dataflow `Decoder` job decodes messages from the PubSub `Raw Topic` to
+  the PubSub `Decoded Topic`
+    - Also checks for existence of `document_id`s in
```

Comment:
We've got a bunch of optional pipeline family features these days, such as debug topics, aet topics, and deduplication (which was originally not optional). Structured is technically not using dedup anymore, though having it in the diagram is illustrative.